app/jobs/reports/identity_reuse_report.rb

```diff
@@ -0,0 +1,51 @@
+require 'identity/hostdata'
+
+module Reports
+  class IdentityReuseReport < BaseReport
+    REPORT_NAME = 'credential-reuse-report'.freeze
+
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      total_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
+    )
+
+    def perform(_date)
+      report = {
+        total_ial1_identity_counts: ial1_identity_counts,
+        total_unique_ial1_identity_counts: unique_ial1_identity_counts,
+        total_ial2_identity_counts: ial2_identity_counts,
+        total_unique_ial2_identity_counts: unique_ial2_identity_counts,
+      }
+      track_report_data_event('Report Identity Reuse Counts', report)
+      save_report(REPORT_NAME, report.to_json, extension: 'json')
+    end
+
+    private
+
+    def ial1_identity_counts
+      transaction_with_timeout do
+        return Db::Identity::TotalIdentitiesPerIalCount.call(1)
```

Comment:
1. `return` is not necessary here
2. let's make `ial` a keyword argument so it's a little clearer?

Reply:
omg lol on the return.  i had a bug with the counts and i threw that in because i thought something wasn't working right!